net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -3,6 +3,7 @@
     <General VisibleName="General" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/general/index" order="1"/>
     <RIP VisibleName="RIP" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/rip/index" order="10" />
     <OSPF VisibleName="OSPF" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf/index" order="20" />
+    <OSPF6 VisibleName="OSPFv3" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/ospf6/index" order="25" />
```

Comment:
@mimugmail Would not use this for OSPF - it is too hard to recognise in a small size like the menu.